rpm/user-managerd.spec

```diff
@@ -65,6 +69,8 @@ make %{?_smp_mflags}
 
 mkdir -p %{buildroot}%{_datadir}/user-managerd/remove.d
 mkdir -p %{buildroot}%{_datadir}/user-managerd/create.d
+mkdir -p %{buildroot}%{_datadir}/user-managerd/session.d
```

Comment:
I'm also wondering if these names should have some symmetry, like `pre-switch.d` and `after-switch.d`. Does that make sense to you?

Reply:
It does not relate to the user switch operation. User-managerd does not detect user change on boot up now.

Reply:
Ok. Maybe it should be `session-start.d` instead. I'm just thinking what this "session" here means. Existing create and remove are clearly actions (verbs) and relate to users (a kind of default for user-managerd), so somehow having word session to differentiate this from user scripts and then have the action there as well would make sense to me. Anyway, after the directory is in use, it's a lot more work to rename it so it'd be good to at least have a reasonable name there.

Similarly that other one could also have the word session so it's perhaps a bit more clear, so `pre-session-switch.d` maybe.